src/less/variables/colors.less

```diff
@@ -32,7 +32,7 @@
 
 /* Main Colors */
 // @orange-main:                               #ee7023; // Not used
-// @orange-headline:                           #f35b1c; // WCAG // Not used
+@orange-headline:                           #f35b1c; // WCAG // Not used
```

Comment:
It's being used now
```suggestion
@orange-headline:                           #f35b1c; // WCAG
```

Reply:
Yes, it is being used now. But only temporarily. So can remove it for now